src/monodroid/jni/monodroid-glue.cc

```diff
@@ -2635,6 +2641,27 @@ MonodroidRuntime::get_java_class_name_for_TypeManager (jclass klass)
 	return ret;
 }
 
+void
+MonodroidRuntime::monodroid_javasystem_loadLibrary (const char *libname) noexcept
+{
+	JNIEnv *env = osBridge.ensure_jnienv ();
+	jmethodID loadLibraryID = env->GetStaticMethodID (mono_android_Runtime, "loadLibrary", "(Ljava/lang/String;)Z");
+	if (loadLibraryID == nullptr) {
+		log_error (LOG_DEFAULT, "Failed to look up the mono.android.Runtime.loadLibrary Java method");
+		return;
```

Comment:
Should this be an "abort the world" scenario?  If we can't find `Runtime.loadLibrary()` something is FUBAR.